src/features/Main/Sidebar/index.jsx

```diff
@@ -0,0 +1,74 @@
+import { AppContext } from '@edx/frontend-platform/react';
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import './index.scss';
+
+export const Sidebar = ({ onClickTab, activeTab }) => {
+  const handleTabClick = (tabName) => {
+    onClickTab(tabName);
+  };
+  const { config } = useContext(AppContext);
+  return (
+    <header className="vertical-nav">
+      <div className="logo">
+        <a href={`${config.BASE_URL}`}>
+          <HomeIcon />
+        </a>
+      </div>
+      <nav className="hidden nav-menu">
+        <ul className="nav-links">
+          <li>
+            <button
+              type="button"
+              className={`${activeTab === 'students' ? 'active' : ''} backbutton`}
+              aria-current="page"
+              onClick={() => handleTabClick('students')}
+            >
+              <span className="icon">
+                <StudentsIcon />
+              </span>
+              <span className="nav-text">Students</span>
+            </button>
+          </li>
+          <li>
+            <button
+              type="button"
+              className={`${activeTab === 'instructors' ? 'active' : ''} backbutton`}
+              aria-current="page"
+              onClick={() => handleTabClick('instructors')}
+            >
+              <span className="icon">
+                <StudentsIcon />
+              </span>
+              <span className="nav-text">Instructors</span>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </header>
+  );
+};
+
+const StudentsIcon = () => (
+  <svg xmlnsXlink="http://www.w3.org/1999/xlink" className="icon-24" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could we move all the svg to assets in a folder called `images` and then call them using `<img src="path/to/your/file.svg">`.

Reply:
@anfbermudezme got it!